python/strings_udf/strings_udf/tests/test_string_udfs.py

```diff
@@ -39,14 +40,27 @@ def get_kernel(func, dtype, size):
     @cuda.jit(
         sig, link=[strings_udf.ptxpath], extensions=[str_view_arg_handler]
     )
-    def kernel(input_strings, output_col):
+    def string_view_kernel(input_strings, output_col):
+        # test the string function with a string_view as input
         id = cuda.grid(1)
         if id < size:
             st = input_strings[id]
             result = func(st)
             output_col[id] = result
 
-    return kernel
+    @cuda.jit(
+        sig, link=[strings_udf.ptxpath], extensions=[str_view_arg_handler]
+    )
+    def udf_string_kernel(input_strings, output_col):
+        # test the string function with a udf_string as input
+        id = cuda.grid(1)
+        if id < size:
+            st = input_strings[id]
+            st = sv_to_udf_str(st)
```

Comment:
There should only be one of these now and should never have been two (likely a copy paste error). I'm not sure how numba resolves two functions that have the same name, I don't _think_ it's by the name, my guess is that since both copies of it didn't do anything except `pass` that their bytecode or whatever numba uses to do its lookup was similar enough that it worked. 

In any case, there's only one now and it's defined in `_testing`.